configs/gtp-amcts.cfg

```diff
@@ -17,6 +17,10 @@ useGraphSearch1 = true
 # This is set to 6 in the original gtp_example.cfg but A-MCTS only supports numSearchThreads = 1
 numSearchThreads = 1
 
+# conservativePass is "a good idea when using [the bot] for analysis or playing
```

Comment:
Can you remind me what the purpose of this file is? Are we trying to match settings to what we use during AMCTS victim-play? Is there a concrete use case for this? (Trying to work out if this flag makes sense given that use case.)

Reply:
We use this for GTP evals and transfer experiments, and in these experiments we're always using Tromp-Taylor